### DISCUSSION TITLE
Import Confluence in existing Wiki.js

### DISCUSSION THREAD

**Discussion by DMW007** [requarks/wiki#4192] [Help / Questions]
Hello,

I have an existing Confluence installation with multiple spaces. Since Confluence stops it's on prem and will be cloud only in the feuture, I want to switch to an self hosted FOSS alternative. Wiki.js seems good for this, the only thing which prevents me from migrating is that the confluence import is not supported yet. It's on the [roadmap for v3](https://docs.requarks.io/en/releases/roadmap). 

I'd like to setup a wiki.js installation and document new content there, also to get more familiar with the system. Would it be possible to import my confluence content into an existing wiki.js wiki which itself has already some content? Or is this not possible, because the import will create a fresh wiki or something like that?

I want to shutdown Confluence when the import works, so to setup wiki.js now it would be a requirement to me that I can merge the pages which I'm creating now with those from the (later done) Confluence import.

**Comment by yrozen-synamedia** [comment]
Hi,
@DMW007  did you received any update on you question?

**Reply by DMW007** [reply]
Hi,
as you can see in the thread, unfortunately not. It's on the roadmap for 3.x while there is currently a beta for 3.0 with ETA H2/2022.
So I fear that this won't be implemented in 2022 too. Maybe there is a way to do this [the HTML export](https://confluence.atlassian.com/doc/export-content-to-word-pdf-html-and-xml-139475.html).

**Reply by rmdes** [reply]
Also interested on this... for the exact same reason

**Comment by rmdes** [comment]
> Hello,
> 
> I have an existing Confluence installation with multiple spaces. Since Confluence stops it's on prem and will be cloud only in the feuture, I want to switch to an self hosted FOSS alternative. Wiki.js seems good for this, the only thing which prevents me from migrating is that the confluence import is not supported yet. It's on the [roadmap for v3](https://docs.requarks.io/en/releases/roadmap). 
> 
> I'd like to setup a wiki.js installation and document new content there, also to get more familiar with the system. Would it be possible to import my confluence content into an existing wiki.js wiki which itself has already some content? Or is this not possible, because the import will create a fresh wiki or something like that?
> 
> I want to shutdown Confluence when the import works, so to setup wiki.js now it would be a requirement to me that I can merge the pages which I'm creating now with those from the (later done) Confluence import.

I will have a look to see if the current import feature is able to work with a html export of confluence.

**Comment by jkour** [comment]
@DMW007 

Have a look at this tool: https://apps.kouraklis.com/conflux/

It has a free version
